src/java.base/share/classes/java/lang/VersionProps.java.template

```diff
@@ -119,6 +123,9 @@ class VersionProps {
         props.put("java.vendor", VENDOR);
         props.put("java.vendor.url", VENDOR_URL);
         props.put("java.vendor.url.bug", VENDOR_URL_BUG);
+
+        // In case VM is not yet set this property
+        props.putIfAbsent("java.vm.vendor", VENDOR_VM);
```

Comment:
Can this system property just be set here for simplicity?  The value of  `java.vendor` and `java.vm.vendor` system property are the same.

The VM does not need this value until VM initialization completes.  It's only needed by StatSampler (jstat).

Reply:
In fact, VM no longer sets this property.   This should be updated to simply do `props.put`.

Reply:
Is it part of the implementation specification?   If not, I would expect other VM implementations should be updated for this release.